packages/ui-tree-browser/src/TreeBrowser/README.md

```diff
@@ -381,6 +381,89 @@ class Example extends React.Component {
 render(<Example/>)
 ```
 
+### Change the order of appearance of items and collection
+
+By default, the order of collections and items depends on the order of `collections` and `items` array. We can override it by add new `sortOrder` comparision function.
+
+---
+
+**NOTE**
+
+This works with all collections and items of the TreeBrowser.
+
+---
+
+```js
+---
+example: true
+render: false
+---
+class Example extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      size: 'medium'
+    }
+
+    this.sizes = ['small', 'medium', 'large']
+  }
+
+  handleSizeSelect = (e, size) => {
+    this.setState({ size })
+  };
```

Comment:
For this example the size setters don't seem relevant, this is already displayed in another example. If you would like to make it interactive, I suggest having a "Sort collections alphabetically" checkbox where you can turn the sorting on and off, and that would display what this property does very nicely :)